templates/directorcontrol/globalsearchtagblacklist.html

```diff
@@ -0,0 +1,38 @@
+$def with (tags, taglist)
+$:{RENDER("common/stage_title.html", ["Edit Global Community Tagging Blacklist", "Director Control Panel"])}
+
+<div class="content form clear">
+  <form method="POST" class="form skinny clear" action="/directorcontrol/globalsearchtagblacklist">
+    <h3>Current Global Community Tagging Blacklist</h3>
+    <table>
+      <thead>
+        <tr>
+          <th width="50%">Added By</th>
+          <th width="50%">Tag</th>
+        </tr>
+      </thead>
+      <tbody>
+        $for entry in tags:
+          <tr>
+            <td style="text-align:center">$entry['login_name']</td>
+            <td style="text-align:center">$entry['title']</td>
+          </tr>
+      </tbody>
+    </table>
+    $:{CSRF()}
+    <h3>Global Community Tagging Blacklist Information</h3>
+    <p>
+      Tags added here will not be able to be added to the list of search tags for an item,
+      except by the owner of the item in question, or Weasyl staff. In addition to blocking individual tags, you can
+      forbid tag patterns by using <code>*</code> as a wildcard. Wildcard entries must have at least 
+      two additional characters. For example, <code>sun*</code> matches <em>sunrise</em> and <em>sunset</em>; 
+      <code>*_pen</code> matches <em>fountain_pen</em> and <em>ballpoint_pen</em>.
+    </p>
+
+    <label>Add or Remove Tags:</label><br />
+    <textarea id="tags-textarea" name="tags">${taglist}</textarea>
```

Comment:
Any reason it’s `${' '.join(tags)}` in the other template and `${taglist}` here?